### DISCUSSION TITLE
Episode 192

### DISCUSSION THREAD

**Discussion by writing-shed** [pointfreeco/swift-composable-architecture#3432] [General]
This episode stops streaming for me after some variable time, usually after about 5mins. This has happened on two networks. Anyone else seeing this?

**Comment by mbrandonw** [comment]
Hi @writing-shed, sorry I'm not able to reproduce this:

![image](https://github.com/user-attachments/assets/137e3528-0140-4b49-8ab7-d49641a52e5d)

Is this the only episode that you are having problems? Is there any other information you can share?
